### DISCUSSION TITLE
Which column am I in?

### DISCUSSION THREAD

**Discussion by InjAnnuity1** [anvilistas/tabulator#100] [Q&A]
In the Anvil Tabulator Example, the Anvil cell component is used in just one column, so it can just assume that it knows its column name (e.g., `'col'`).

In practice, it might be used from any number of different columns.  In that case, how would any particular instance know _which_ column had invoked it?

I didn't find anything in `**properties` that would tell it.

**Comment by lewysigns** [comment]
Here is a method I add to the tabulator ```cell_click``` event:

```python
  def cell_click(self,cell, **event_args):
    field    = cell.get_field() #Column Name
    row      = cell.get_row()  #Tabulator Row
    row_data = row.get_data() #Dictionary of Tabulator Row
    self.edit_map[field](row,row_data)
```

**Comment by InjAnnuity1** [comment]
Resolved in v2.2.10.  The cell has been added to `**properties`.

**Reply by lewysigns** [reply]
It was an easy [update](https://github.com/s-cork/Tabulator/pull/101#issue-1637752818)! :)
